### PR TITLE
Item Schema

### PR DESCRIPTION
**Pull request by diversifiedmechanic**
I should have everything set up to start using this schema.

### CODE REVIEW

**Comment by diversifiedmechanic** [comment]
@ZacharyAb or @Lept14 or @naszzz want to give me a code review?

File: database/Item.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  itemPrice: Number,
+  itemPicture: String,
+  itemShippingPrice: Number,
+  sellerPicture: String,
+  sellerName: String,
+  sellerCountry: String,
+  sellerTotalSales: Number,
+  sellerJoinDate: Date,
+})
+
```
Comment (ZacharyAb) [inline database/Item.js]: lgtm

File: server/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // this is where my app will start
 const express = require('express');
 require('dotenv').config();
+const db = require('../database');
```
Comment (ZacharyAb) [inline server/index.js]: GJ starting set up with .env

Comment (diversifiedmechanic) [reply server/index.js]: Thanks!  Hopefully it'll save us some time in the future.

**Pull request merged**